package.json

```diff
@@ -77,9 +75,9 @@
     "html5 player"
   ],
   "dependencies": {
-    "@playkit-js/common": "^1.2.13",
-    "@playkit-js/playkit-js-ui": "^0.77.3",
-    "@playkit-js/ui-managers": "^1.3.11",
+    "@playkit-js/common": "1.5.9",
+    "@playkit-js/playkit-js-ui": "0.78.0",
```

Comment:
why is ui in dependencies and not dev dependencies ? why depend on ui and not on the player ?

Reply:
transcript plugin takes SCSS variables from UI repo
https://github.com/kaltura/playkit-js-transcript/blob/master/src/variables.scss#L1